packages/web/src/components/nav/desktop/AccountDetails.tsx

```diff
@@ -1,66 +1,130 @@
+import { useIsManagedAccount } from '@audius/common/hooks'
 import { FeatureFlags } from '@audius/common/services'
 import { accountSelectors } from '@audius/common/store'
-import { Text } from '@audius/harmony'
+import { Box, Flex, Text, useTheme } from '@audius/harmony'
 
 import { AvatarLegacy } from 'components/avatar/AvatarLegacy'
 import { TextLink, UserLink } from 'components/link'
 import { useFlag } from 'hooks/useRemoteConfig'
 import { useSelector } from 'utils/reducer'
 import { SIGN_IN_PAGE, profilePage } from 'utils/route'
 
-import styles from './AccountDetails.module.css'
 import { AccountSwitcher } from './AccountSwitcher/AccountSwitcher'
 
 const { getAccountUser } = accountSelectors
 
 const messages = {
   haveAccount: 'Have an Account?',
+  managedAccount: 'Managed Account',
   signIn: 'Sign in'
 }
 
 export const AccountDetails = () => {
   const account = useSelector((state) => getAccountUser(state))
-  const { isEnabled: isManagerModeEnabled } = useFlag(FeatureFlags.MANAGER_MODE)
+  const { color } = useTheme()
+  const { isEnabled: isManagerModeEnabled = false } = useFlag(
+    FeatureFlags.MANAGER_MODE
+  )
+  const isManagedAccount = useIsManagedAccount() && isManagerModeEnabled
 
   const profileLink = profilePage(account?.handle ?? '')
 
   return (
-    <div className={styles.userHeader}>
-      <div className={styles.accountWrapper}>
-        <AvatarLegacy userId={account?.user_id} />
-        <div className={styles.userInfo}>
-          {account ? (
-            <>
-              <UserLink
-                textVariant='title'
-                size='s'
-                strength='weak'
-                userId={account.user_id}
-                badgeSize='xs'
-              />
-              <TextLink
-                size='xs'
-                to={profileLink}
-              >{`@${account.handle}`}</TextLink>
-            </>
-          ) : (
-            <>
-              <Text variant='body' size='s' strength='strong'>
-                {messages.haveAccount}
-              </Text>
-              <TextLink
-                to={SIGN_IN_PAGE}
-                variant='visible'
-                size='xs'
-                strength='weak'
-              >
-                {messages.signIn}
-              </TextLink>
-            </>
-          )}
-        </div>
-        {isManagerModeEnabled && account ? <AccountSwitcher /> : null}
-      </div>
-    </div>
+    <Flex direction='column' pb='unit5' w='100%'>
+      {isManagedAccount ? (
+        <Box pv='xs' ph='m' backgroundColor='accent'>
+          <Text variant='label' size='xs' color='staticWhite'>
+            {messages.managedAccount}
+          </Text>
+        </Box>
+      ) : null}
+      <Flex
+        pt='l'
+        pr='s'
+        pb='s'
+        pl='m'
+        css={{
+          ...(isManagedAccount
```

Comment:
could we just do `backgroundColor: `${color.background.accent}08`` on the flex?

Reply:
we could also consider something like this: https://polished.js.org/

Reply:
I think I tried it and it didn't work for some reason, but I'll double check.
I think polished or a tool like it should, at most, be incorporated into a toolset in harmony. We have minimal needs for things like lightening the current color by X% on hover, etc. So I'd be wary of introducing another code pattern and library just for the sake of a few common style manipulations.

What we _could_ do though is make this styling a reusable utility. I thought about that and I think the only reason I didn't do it was two usages in the same directory didn't seem like enough motivation yet.

Reply:
Alright so the main issue with using background is that we'd have to modify the color to get it the desired 3% opacity. We would need a tool like polished (or to write our own color manipulation functions) to accomplish that.
In lieu of adding that at this point, [I moved this into a helper style util](https://github.com/AudiusProject/audius-protocol/pull/8594/commits/427eb77d90cd22eb0b37e800b5c78540e5ff1822) that can be spread into the `css` prop to make it clear what's being done (adding a background overlay) and make it reusable.

But I think we should come back around to this and decide how we want to handle the color manipulation specifications we have for harmony components  ('darken by this amount on press', etc).

Additionally the [relative colors spec](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_colors/Relative_colors) looks promising, but at first glance I don't think there's a polyfill and it's not widely supported yet: